Library/Formula/subversion.rb

```diff
@@ -70,6 +70,18 @@ class Subversion < Formula
   end
 
   def install
+    # OS X's Python is built universally and can't link with Homebrew's deps
+    # unless Homebrew's deps are universal as well.
+    # https://github.com/Homebrew/homebrew-versions/issues/777
+    if build.with?("python") && !File.exist?(HOMEBREW_PREFIX/"bin/python")
+      unless build.universal?
+        raise <<-EOS.undent
+          You must build subversion17 --universal unless Homebrew's
```

Comment:
Oops. Someone can kick me for forgetting to change this line when I copied it across :smile_cat: